doc/data/messages/s/simplifiable-condition/bad.py

```diff
@@ -0,0 +1,3 @@
+APPLES = 100
+
+bool(APPLES or False)  # [simplifiable-condition]
```

Comment:
```suggestion
def has_apples(apples) -> bool:
    return bool(apples or False)  # [simplifiable-condition]
```

When there is a boolean message I think it's better if there is not too much information about the non problematic portion of the conditional, because if we have a constant it means we could further simplify (100 is truethey, we could just return True)